src/sendalert.cpp

```diff
@@ -86,13 +86,19 @@ void ThreadSendAlert()
     //  2000 for longer invalid proof-of-work chain 
     //  Higher numbers mean higher priority
     //  4000 or higher will put the RPC into safe mode
-    alert.nPriority     = 5000;
+    alert.nPriority     = 1500;
     alert.strComment    = "";
-    alert.strStatusBar  = "URGENT: Upgrade required: see https://z.cash";
-    alert.strRPCError   = "URGENT: Upgrade required: see https://z.cash";
+    alert.strStatusBar  = "Your client is out of date and potentially vulnerable to denial of service. Please update to the most recent version of Zcash (1.0.5). More info at: https://z.cash/support/security.html";
+    alert.strRPCError   = "Your client is out of date and potentially vulnerable to denial of service. Please update to the most recent version of Zcash (1.0.5). More info at: https://z.cash/support/security.html";
 
     // Set specific client version/versions here. If setSubVer is empty, no filtering on subver is done:
     // alert.setSubVer.insert(std::string("/MagicBean:0.7.2/"));
+	alert.setSubVer.insert(std::string("/MagicBean:1.0.3/"));
```

Comment:
@Maroc-OS This PR is more about keeping a historical record of alerts that have been sent out.